Model/Sync/Catalog/Processor.php

```diff
@@ -165,6 +186,8 @@ public function process($forceSyncProducts = [], $order = null, $storeIds = [])
                             echo 'Catalog sync is disabled for store - ' .
                                 $this->coreConfig->getStoreName($storeId) . PHP_EOL;
                         }
+                    }
+                    if ($disabled) {
```

Comment:
If the catalog sync is disabled but it's a sync following order - the sync should move forward.
In this case, the product sync won't move forward and the order sync will fail.

Reply:
Hi @LiorGingi ,

If the catalog sync is called by an order, the $order object will have some value in it. So in that case the "if (!$order && !$this->coreConfig->isCatalogSyncActive()) {" the condition will not be triggered. This condition will be hit only if the $order object is an empty one (which means it is a normal product sync)

Referring to the above code, $disabled flag will be true only in 2 cases.

1.  if Yotpo module is disabled for a particular store view
2. If the product sync is not called from an order AND the product sync is disabled

Please correct me if I am wrong here.